server/service/src/stocktake/update.rs

```diff
@@ -886,6 +894,7 @@ mod test {
             stocktake_line.sell_price_per_pack.unwrap()
         );
         assert_eq!(stock_line.note, stocktake_line.note);
+        assert_eq!(stock_line.name_id, None);
```

Comment:
would be good to also have a test that links a supplier name in

Reply:
[2e1c1ac](https://github.com/openmsupply/open-msupply/pull/1044/commits/2e1c1ace4742de85361e40e3522665ab6a2212b1)